src/cozmo/run.py

```diff
@@ -103,20 +113,50 @@ class IOSConnector(DeviceConnector):
 
     An instance of this class can be passed to the ``connect_`` prefixed
     functions in this module.
+
+    Args:
+        serial (string): Serial number of the device to connect to.
+            If None, then connect to the first available iOS device running
+            the Cozmo app in SDK mode.
     '''
-    def __init__(self, **kw):
+    def __init__(self, serial=None, **kw):
         super().__init__(**kw)
         self.usbmux = None
+        self._connected = set()
+        self.serial = serial
```

Comment:
I don't really see how it's going to be confused in the context its in, nor with the docstring that makes it explicit